examples/npm_package/packages/pkg_a/BUILD.bazel

```diff
@@ -1,16 +1,29 @@
+load("@aspect_bazel_lib//lib:jq.bzl", "jq")
 load("@aspect_rules_js//npm:defs.bzl", "npm_package")
 load("@aspect_rules_js//js:defs.bzl", "js_library")
 load("@npm//:defs.bzl", "npm_link_all_packages")
 
 npm_link_all_packages(name = "node_modules")
 
+jq(
```

Comment:
Do you think this will be common enough that it's worth wrapping in a macro supplied by rules_js?

Reply:
that's what I'm wondering here - I secretly sent this as a strawman API proposal. At first I thought it could look like

```
npm_package(
    ...
    package_json = "package.json",
    version_stamp_key = "STABLE_BUILD_VERSION",
)
```

but I think it might be load-bearing in the dependency graph that the package.json comes along with the js_library so fields like `main` can be read at the use site without having to go through an npm_package?

Reply:
the other problem with a macro is which file to put it in it since it adds a dependency on a jq toolchain which not all rules_js users have setup... I think this means it must be a new starlark file rather than adding a `load` statement to an existing one

Reply:
oh I'm wrong there, the toolchain is only needed at analysis time, not loading time, so putting into existing file is fine